scripts/setupv2.sh

```diff
@@ -96,24 +98,54 @@ done
 lnImplementation=""
 
 while true; do
-  read -p "Which lightning implementation do you want to tunnel? Supported are LND and CLN for now ⚡️: " answer
-
-  case $answer in
-  lnd | LND*)
-    echo "> Setting up Tunneling for LND on port 9735 "
-    echo
-    lnImplementation="lnd"
-    break
-    ;;
-
-  cln | CLN*)
-    echo "> Setting up Tunneling for CLN on port 9735 "
-    echo
-    lnImplementation="cln"
-    break
-    ;;
+  echo "Which lightning implementation do you want to tunnel?"
+  echo "    1) LND"
+  echo "    2) CLN (Core Lightning)"
+  if [ $litpossible -eq 1 ]; then
+    echo "    3) LIT"
```

Comment:
```suggestion
    echo "    3) LIT (integrated mode)"
```

shall we make it clear that this requires LiT running in "integrated mode"?